pkg/apis/kops/v1alpha2/instancegroup.go

```diff
@@ -172,6 +172,11 @@ type InstanceGroupSpec struct {
 	CompressUserData *bool `json:"compressUserData,omitempty"`
 	// InstanceMetadata defines the EC2 instance metadata service options (AWS Only)
 	InstanceMetadata *InstanceMetadataOptions `json:"instanceMetadata,omitempty"`
+	// UpdatePolicy determines the policy for applying upgrades automatically.
```

Comment:
Add a note here that this overrides the "spec.updatePolicy" field in the _Cluster_ spec.